source/distributed/tria.cc

```diff
@@ -1309,7 +1309,8 @@ namespace parallel
       triangulation_has_content (false),
       connectivity (nullptr),
       parallel_forest (nullptr),
-      cell_attached_data ({0, 0, 0, {}})
+      cell_attached_data ({0, 0, 0, {}}),
+      local_quadrant_cell_relations ()
```

Comment:
You don't actually need this line -- the variable is a `std::vector` which has a default constructor that would simply be called if you didn't say anything here.